sources/HUBComponentModelBuilderImplementation.m

```diff
@@ -48,13 +48,15 @@ @interface HUBComponentModelBuilderImplementation ()
 @property (nonatomic, strong, nullable) HUBComponentTargetBuilderImplementation *targetBuilderImplementation;
 @property (nonatomic, strong, readonly) NSMutableDictionary<NSString *, HUBComponentModelBuilderImplementation *> *childBuilders;
 @property (nonatomic, strong, readonly) NSMutableArray<NSString *> *childIdentifierOrder;
+@property (nonatomic, strong, nullable) NSMutableDictionary<NSString *, NSMutableArray<id<HUBComponentModelBuilder>> *> *childBuildersByGroupIdentifier;
```

Comment:
Since this is created in the initializer, it should be `readonly` instead of `nullable`.